volcli/volcli.go

```diff
@@ -350,15 +377,31 @@ func volumeForceRemove(ctx *cli.Context) (bool, error) {
 		return true, err
 	}
 
-	cfg, err := config.NewClient(ctx.GlobalString("prefix"), ctx.GlobalStringSlice("etcd"))
+	request := config.Request{
+		Policy: policy,
+		Volume: volume,
+	}
+
+	content, err := json.Marshal(request)
 	if err != nil {
 		return false, err
 	}
 
-	if err := cfg.RemoveVolume(policy, volume); err != nil {
+	resp, err := http.Post(fmt.Sprintf("http://%s/volumes/removeforce", ctx.String("volmaster")), "application/json", bytes.NewBuffer(content))
+	if err != nil {
 		return false, err
 	}
 
+	qualifiedVolume := strings.Join([]string{policy, volume}, "/")
+
+	if resp.StatusCode == 404 {
+		return false, errored.Errorf("Volume %v no longer exists.", qualifiedVolume)
+	}
+
+	if resp.StatusCode != 200 {
+		return false, errored.Errorf("Volume %v Response Status Code was %d, not 200: %v", qualifiedVolume, resp.StatusCode, resp.Status)
```

Comment:
should we dump the resp body on stderr here? I think this is true for other places as well. I see we dump the body only in few cases, but I feel we should dump it always in case of non 200 (or 404) return, usually the body will have more informative cues on the failure.